extensions/AbuseFilter/AbuseFilterVariableHolder.php

```diff
@@ -314,22 +314,7 @@ function compute( $vars ) {
 					break;
 				}
 
-				$dbr = wfGetDB( DB_SLAVE );
-				$res = $dbr->select( 'revision',
-					'DISTINCT rev_user_text',
-					array(
-						'rev_page' => $title->getArticleId(),
-						'rev_timestamp<' . $dbr->addQuotes( $dbr->timestamp( $cutOff ) )
-					),
-					__METHOD__,
-					array( 'ORDER BY' => 'rev_timestamp DESC', 'LIMIT' => 10 )
-				);
-
-				$users = array();
-				foreach( $res as $row ) {
-					$users[] = $row->rev_user_text;
-				}
-				$result = $users;
+				$result = $this->getRecentAuthorsNames( $title->getArticleId(), $cutOff );
```

Comment:
Let's annotate our changes to core + WMF extensions with ticket ID and description